app/Http/Controllers/CampaignsController.php

```diff
@@ -58,6 +58,11 @@ public function index(Request $request)
             }
         }
 
+        // Apply scope for campaigns including specified causes:
+        if (isset($filters['causes'])) {
```

Comment:
This is super nitpicky, but should this be `cause` (so, that it's singular like `?filter[id]=1,2,3`)?

Reply:
Ya know what, I may have misread your comment in the Slack thread. Will update! This would also be consistent with the [`tag` filter](https://github.com/DoSomething/rogue/blob/cb4da53aba1f74c7f77c722b3085b7f05c186bdf/app/Http/Controllers/PostsController.php#L89-L92)

Reply:
Updated in https://github.com/DoSomething/rogue/pull/985/commits/2efce6c03f0c82eb946017e85d6001836ddfdc8f